.travis.yml

```diff
@@ -26,7 +26,7 @@ matrix:
     - php: hhvm
       env: DB=mysqli
   allow_failures:
-    - php: 7.0
+    - env: DB=sqlite3
```

Comment:
@michaelcullum we can either keep it as an allowable failure like this, or I think better to just remove it from the matrix